enterprise/cypher/spec-suite-tools/pom.xml

```diff
@@ -199,6 +199,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scalap</artifactId>
+      <version>2.12.5</version>
```

Comment:
The rest of the project is using Scala `2.11.12`. What is the reason this needs to be 2.12? Maybe we should write a comment about that. Or if the latest 2.11 is fine we should use `${scala.version}`.